_notebooks/2026-accelerate-cfd.md

```
@@ -0,0 +1,210 @@
+The Kármán vortex street is clearly visible in both the pressure and velocity fields. At t ≈ 100 s the wake already shows the characteristic lateral modulation; at t ≈ 200 s (end of the first CFD block) the alternating high- and low-velocity structures extend several diameters downstream. Low-pressure regions (blue) alternate on either flank of the cylinder as vortices are shed and convected.
+
+This periodicity — and the compactness of its POD representation — is what makes the cylinder the canonical benchmark for the adaptive framework: **5 modes suffice to capture over 90% of the fluctuating kinetic energy**, so the LSTM operates on a 5-dimensional coefficient space.
+
+<div align="center">
+<table>
+<tr>
+  <th align="center">Pressure <i>p</i> — t ≈ 100 s</th>
+  <th align="center">Velocity |<b>U</b>| — t ≈ 100 s</th>
+</tr>
+<tr>
+  <td align="center"><img src="../assets/img/figs/cap3/cilindro_p_t0.png" width="340"></td>
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)